src/Symfony/Bundle/TwigBundle/Tests/DependencyInjection/Compiler/TwigEnvironmentPassTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\TwigBundle\Tests\DependencyInjection\Compiler;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Bundle\TwigBundle\DependencyInjection\Compiler\TwigEnvironmentPass;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+
+class TwigEnvironmentPassTest extends TestCase
+{
+    /**
+     * @var ContainerBuilder
+     */
+    private $builder;
+    /**
+     * @var Definition
+     */
+    private $definition;
+    /**
+     * @var TwigEnvironmentPass
+     */
+    private $pass;
+
+    protected function setUp()
```

Comment:
we don't need this method IMHO, better remove it and the corresponding properties

Reply:
Done :)